tests/test_resampler.py

```diff
@@ -223,6 +277,10 @@ def test_resampling_function_name_value() -> None:
     assert ResamplingFunction.Last.value == 4
     assert ResamplingFunction.Count.name == "Count"
     assert ResamplingFunction.Count.value == 5
+    assert ResamplingFunction.First.name == "First"
+    assert ResamplingFunction.First.value == 6
+    assert ResamplingFunction.Coalesce.name == "Coalesce"
+    assert ResamplingFunction.Coalesce.value == 7
```

Comment:
Is this test even needed, i.e. does something rely on the mapping between names and values or the name strings (e.g. the python code)?

Reply:
Unfortunately, an Enum in Rust is not an Enum in Python after creating the bindings. Therefore, we have to rebuild it in a way to have the same behavior an Enum would have. This is tested here.